src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -78,17 +63,23 @@ def openvino_compile_cached_model(cached_model_path, *example_inputs):
         om.inputs[idx].get_node().set_partial_shape(PartialShape(list(input_data.shape)))
     om.validate_nodes_and_infer_types()
 
-    core.set_property({'CACHE_DIR': cache_root_path() + '/blob'})
+    core.set_property({'CACHE_DIR': _get_cache_dir(options) + '/blob'})
+
+    config = _get_config(options)
+
+    if config:
+        for config_name, value in config.items():
+            core.set_property({config_name: value})
```

Comment:
I don't think you need to `set_property` one by one, you could pass config to compile_model

Reply:
agree, similarly for `CACHE_DIR`

Reply:
Done